docs/releasing.md

```diff
@@ -40,13 +40,11 @@ Note: actively undergoing changes by @exterkamp and @egsweeny.
 1. Release mgr tells the _LH public_ Hangout chat about the new version.
 1. V & Kayce write and publish the [/updates](https://developers.google.com/web/updates/) blog post
```

Comment:
```suggestion
1. Release mgr optionally write and publish the [/updates](https://developers.google.com/web/updates/) blog post, if appropriate
```

Reply:
Can we remove this? afaik we've never ever done this.